src/main/java/mil/dds/anet/resources/NoteResource.java

```diff
@@ -91,9 +89,41 @@ public Integer deleteNote(@GraphQLRootContext Map<String, Object> context,
   }
 
   private void checkPermission(Note n, final Person user) {
-    if (!n.getAuthorUuid().equals(DaoUtils.getUuid(user)) && !AuthUtils.isAdmin(user)) {
-      throw new WebApplicationException("Only the author or an admin can do this",
-          Status.FORBIDDEN);
+    // Node type is ASSESSMENT
```

Comment:
We don't want to change other note types behavior. So i think we should keep this removed part and add assessments checks as an addition like this:

```
    if (isAuthor || isAdmin) {
      return; // just works as previously
    }
    if ( type == assessment ){
      do additional checks...
      return; 
    }
    throw Forbidden;
```